src/pages/index.js

```diff
@@ -94,6 +139,28 @@ function handleEscape(evt) {
   }
 }
 
+// edit avatar functions
+function handleAvatarSubmit(evt) {
+  evt.preventDefault();
+  const saveButton = evt.submitter;
+  setButtonText(saveButton, true, saveButton.textContent, "Saving...");
+  api
+    .editAvatarInfo(avatarInput.value)
+    .then((data) => {
+      avatarImageElement.src = data.avatar;
+      closeModal(avatarModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(saveButton, false, saveButton.textContent, "Saving...");
```

Comment:
` saveButton.textContent`  is not needed in `finally` because it contains `Saving...`, but you need to return the initial button text back in `finally`